Assets/Scripts/Puzzle/Gate.cs

```diff
@@ -10,15 +11,20 @@ public class Gate : SignalReceiver {
 	[Header("Gate")]
 	[SerializeField] private Animator animator;
 	[SerializeField] private bool panCamera;
-	[SerializeField] private float showDuration = 1f;
+	[SerializeField, EnableIf(nameof(panCamera))]
+	private float showDuration = 1f;
+	[SerializeField, EnableIf(nameof(panCamera))]
+	private float showCooldown = 7.5f;
+
+	private static List<Gate> panningQueue = new List<Gate>();
```

Comment:
Using a `Queue<T>` here would be more efficient.

Reply:
Of course that's a thing